libs/ledger-live-common/src/families/celo/js-getFeesForTransaction.ts

```diff
@@ -9,29 +11,120 @@ const getFeesForTransaction = async ({
   account: Account;
   transaction: Transaction;
 }): Promise<BigNumber> => {
-  const { amount } = transaction;
+  const { amount, index } = transaction;
   const kit = celoKit();
 
   // A workaround - estimating gas throws an error if value > funds
-  const value = transaction.useAllAmount
-    ? account.spendableBalance
-    : BigNumber.minimum(amount, account.spendableBalance);
+  let value;
```

Comment:
It would be great to type this var as well - `let value: BigNumber = new BigNumber(0);`